monai/data/utils.py

```diff
@@ -288,7 +289,11 @@ def list_data_collate(batch: Sequence):
         raise TypeError(re_str)
 
 
-def decollate_batch(data: Union[dict, list, torch.Tensor], batch_size: Optional[int] = None) -> List[dict]:
+def decollate_batch(
+    data: Union[dict, list, torch.Tensor],
+    batch_size: Optional[int] = None,
+    copy_non_batch: bool = True,
+) -> List[dict]:
```

Comment:
`) -> List[dict, list, torch.Tensor]:`

Reply:
Sure, updated in the latest commit.
Thanks.